eoscostarica/eosio-dashboard
# Add missing sensors

### Add missing sensors

### What does this PR do?
- Resolve #129 

### Steps to test
1. yarn
2. yarn start
3. Navigate to http://localhost:3000/dashboard/producers
4. Check the producer info

#### CheckList
